brave/brave-core
# Use probi for reports

Resolves https://github.com/brave/brave-browser/issues/948
Resolves https://github.com/brave/brave-browser/issues/1285
Resolves https://github.com/brave/brave-browser/issues/1472

Ledger implementation: https://github.com/brave-intl/bat-native-ledger/pull/86
UI implementation: https://github.com/brave/brave-ui/pull/187

## Submitter Checklist:


## Test Plan:
- clean profile
- enable rewards
- claim grant
- make sure that wallet summary reflects all data correctly

## Reviewer Checklist:
